main.go

```diff
@@ -552,17 +552,19 @@ func main() {
 		handleConfigError(log, true, "ERROR: --repo must be specified")
 	}
 
-	if *flDeprecatedBranch != "" && (*flDeprecatedRev == "" || *flDeprecatedRev == "HEAD") {
+	switch {
```

Comment:
I'm not sure I personally agree with this linter, but it is based on https://go.dev/doc/effective_go#switch (according to https://github.com/go-critic/go-critic/issues/453)

Reply:
Yeah, I don't agree with this guidance as dogmatic.  I prefer the else-if but it's fine.  not a big deal if it shuts up the tool